system-tests/e2e-suite/src/test/scala/com/hotels/styx/proxy/ChunkedUploadSpec.scala

```diff
@@ -203,10 +203,10 @@ class ChunkedUploadSpec extends FunSpec
 
       assert(response.nonEmpty, "\nStyx did not respond.")
       val content = response.get.content().toString(UTF_8)
-      assert(response.get.getStatus == REQUEST_TIMEOUT, s"\nExpecting 408 Request Timeout, but got: \n$response \n\n$content\n\n")
+      assert(response.get.status() == REQUEST_TIMEOUT, s"\nExpecting 408 Request Timeout, but got: \n$response \n\n$content\n\n")
     }
 
-    it("Responds with 400 Bad Request when chunk size 80000000 causes an integer overflow.") {
+    it("Responds with 500 Internal Server Error when chunk size 80000000 causes an integer overflow.") {
```

Comment:
why did 400 became 500 in these tests?

Reply:
Changed it. In the latest Netty versions, there's no longer a limit for max chunk size.